Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meals.yml

```diff
@@ -361,3 +361,193 @@
   - type: Tag
     tags:
     - Fruit
+
+- type: entity
+  name: basic noodles
+  parent: FoodBowlBase
+  id: FoodMealBasicNoodles
+  description: A simple ramen bowl of noodles and broth. Yum!
+  components:
+  - type: FlavorProfile
+    flavors:
+      - finenoodles
+      - richbroth
+  - type: Sprite
+    sprite: _NF/Objects/Consumable/Food/bowl.rsi
+    layers:
+    - state: bowl
+    - state: basic-noodles
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 35
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 15
+        - ReagentId: Flavorol
+          Quantity: 10
+
+- type: entity
+  name: miso noodles
+  parent: FoodBowlBase
+  id: FoodMealMisoNoodles
+  description: 
+  components:
+  - type: FlavorProfile
+    flavors:
+      - finenoodles
+      - richbroth
+      - tangy
+  - type: Sprite
+    sprite: _NF/Objects/Consumable/Food/bowl.rsi
+    layers:
+    - state: bowl
+    - state: miso-noodles
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 35
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 15
+        - ReagentId: Vitamin
+          Quantity: 5
+        - ReagentId: Flavorol
+          Quantity: 10
+        - ReagentId: Protein
+          Quantity: 3
+  - type: Tag
+    tags:
+    - Meat
+
+- type: entity
+  name: shio noodles
+  parent: FoodBowlBase
+  id: FoodMealShioNoodles
+  description: Sea salt broth with corn and seaweed.
```

Comment:
Missing the little "Light!" at the end (the full description being "Sea salt broth with corn and seaweed. Light!")